source/extensions/filters/network/thrift_proxy/conn_manager.h

```diff
@@ -272,6 +273,10 @@ class ConnectionManager : public Network::ReadFilter,
   bool stopped_{false};
   bool half_closed_{false};
   TimeSource& time_source_;
+
+  // The maximum number of requests remaining to be processed on the current connection.
+  uint64_t remaining_streams_{};
```

Comment:
Can we do this the other way around? Having a counter of seen requests and a the limit in a separate var? E.g.:

```
  uint64_t requests_{};
  uint64_t max_requests_{}; 
```

If max_requests_ is 0, there are no limits. I rather avoid depending on `std::numeric_limits<uint64_t>::max()`.

Reply:
I have no objection. The reason for choosing this method is to reduce one `if` judgment. If you use this method, you need to determine whether max_requests is 0, and then determine whether the accumulated request value is greater than max_requests.

Reply:
But it should be ok.